### PR TITLE
Added code for OpenDyslexic Font 

### PR DESCRIPTION
**Pull request by ghost**
OpenDyslexic Font applied when running the extension.

### CODE REVIEW

**Comment by virtadpt** [comment]
Yay!  Thank you!

File: readable_text.js [hunk 1]
```diff
@@ -1,4 +1,7 @@
-javascript: (() => {
```
Comment (Fillyosopher) [inline readable_text.js:1]: Could you clean this up? :)

File: readable_text.js [hunk 2]
```diff
@@ -10,13 +13,18 @@ javascript: (() => {
...
     /* iterating through all children of the parent*/
+
     for (var i = 0; parentElement.childNodes[i] != undefined; i++) {
       /* if the child is a text element*/
       if (parentElement.childNodes[i].nodeName == "#text" &&
         parentElement.childNodes[i].textContent.trim().length != 0
       ) {
-        var recentNode = parentElement.childNodes[i];
+
+        var recentNode = parentElement.childNodes[i]; // We cannot modify the text node but we will modify it's parent element
```
Comment (Fillyosopher) [inline readable_text.js:23]:
```suggestion
        var recentNode = parentElement.childNodes[i]; /* We cannot modify the text node but we will modify it's parent element */
```
Open ended comments break the bookmarklet, even though they work for the extension.

**Review by Fillyosopher — CHANGES_REQUESTED** [review]
Hey, thanks so much for the PR!  I'm thinking about how best to have this feature be optional, but for now I'm willing to approve it with two minor changes.

**Pull request closed without merging**